docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.3 Release Notes
+description: Release notes for NuGet 6.3 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
+ms.author: <MicrosoftAlias>
+ms.date: 8/2/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.3
+
+* Implement "consuming pdbs from packages in PackageReference" - [#11352](https://github.com/NuGet/Home/issues/11352)
+
+* Create experimentation metrics to measure engagement of transitive dependencies - [#10759](https://github.com/NuGet/Home/issues/10759)
+
+* Enhance the experimentation infrastructure in NuGet code to support transitive dependencies - [#10758](https://github.com/NuGet/Home/issues/10758)
+
+* Allow to user to input custom (floating) versions through the PM UI - [#9829](https://github.com/NuGet/Home/issues/9829)
```

Comment:
@martinrrm Do we have a small blurb & screenshot or gif we can include here? I can help coordinate.

Reply:
I recorded some gifs when I closed https://github.com/NuGet/Home/issues/3788. 

Gifs here: https://github.com/NuGet/Home/issues/3788#issuecomment-1198780610

you don't have to reuse them, but maybe you get an idea about what we can show :) 

screen2gif was super easy to configure for this.

Reply:
On that topic, I think our tool misses issues that are just labeled with the milestone.

In particular, https://github.com/NuGet/Home/issues/3788 is not in the list.
We should add it there. It's a top voted NuGet issue.

Reply:
Definitely. Let's link to the original issues where the most community engagement is. (i.e. 3788)

Reply:
I added it as a new entry in Whats New, is this correct?

Reply:
I'd only keep one of them.